code/datums/components/storage/storage.dm

```diff
@@ -115,6 +119,21 @@
 /datum/component/storage/PreTransfer()
 	update_actions()
 
+/datum/component/storage/proc/set_holdable(can_hold_list, cant_hold_list)
+	can_hold_description = generate_hold_desc(can_hold_list)
+
+	can_hold = typecacheof(can_hold_list)
+	cant_hold = typecacheof(cant_hold_list)

```

Comment:
You're passing null into the args here for when it's unchanged but then you make a typecache and assign from that null. You should do nothing if given null